### ISSUE TITLE
Server upgrade database migration error

### ISSUE THREAD

**Issue by Linus4** [seek4science/seek#755]
**Describe the bug**
I'm trying to upgrade a server from version 1.10.3 to 1.11.2. The upgrade fails while running the `docker/upgrade.sh` script.

**To Reproduce**
1. Follow [instructions](https://docs.seek4science.org/tech/docker/docker-compose.html)
2. `docker exec -it seek docker/upgrade.sh` fails with error.

**Expected behavior**
Database upgrade should complete successfully.

**Logs**
```
$ docker exec -it seek docker/upgrade.sh
USING MYSQL
Starting upgrade ...
... trimming old session data ...
... migrating database ...
== 20190329092301 AddTableHumanDiseases: migrating ============================
-- create_table("human_diseases", {:options=>"ENGINE=InnoDB", :force=>:cascade, :id=>:integer})
rake aborted!
StandardError: An error has occurred, all later migrations canceled:

Mysql2::Error: Tablespace '`seek_docker`.`human_diseases`' exists.: CREATE TABLE `human_diseases` (`id` int NOT NULL AUTO_INCREMENT PRIMARY KEY, `title` varchar(255), `doid_id` varchar(255), `created_at` datetime, `updated_at` datetime, `first_letter` varchar(255), `uuid` varchar(255)) ENGINE=InnoDB
/seek/vendor/bundle/ruby/2.6.0/gems/mysql2-0.5.3/lib/mysql2/client.rb:131:in `_query'
/seek/vendor/bundle/ruby/2.6.0/gems/mysql2-0.5.3/lib/mysql2/client.rb:131:in `block in query'
/seek/vendor/bundle/ruby/2.6.0/gems/mysql2-0.5.3/lib/mysql2/client.rb:130:in `handle_interrupt'
/seek/vendor/bundle/ruby/2.6.0/gems/mysql2-0.5.3/lib/mysql2/client.rb:130:in `query'
/seek/vendor/bundle/ruby/2.6.0/gems/activerecord-5.2.4.6/lib/active_record/connection_adapters/abstract_mysql_adapter.rb:187:in `block (2 levels) in execute'
/seek/vendor/bundle/ruby/2.6.0/gems/activesupport-5.2.4.6/lib/active_support/dependencies/interlock.rb:48:in `block in permit_concurrent_loads'
/seek/vendor/bundle/ruby/2.6.0/gems/activesupport-5.2.4.6/lib/active_support/concurrency/share_lock.rb:187:in `yield_shares'
/seek/vendor/bundle/ruby/2.6.0/gems/activesupport-5.2.4.6/lib/active_support/dependencies/interlock.rb:47:in `permit_concurrent_loads'
/seek/vendor/bundle/ruby/2.6.0/gems/activerecord-5.2.4.6/lib/active_record/connection_adapters/abstract_mysql_adapter.rb:186:in `block in execute'
/seek/vendor/bundle/ruby/2.6.0/gems/activerecord-5.2.4.6/lib/active_record/connection_adapters/abstract_adapter.rb:581:in `block (2 levels) in log'
/seek/vendor/bundle/ruby/2.6.0/gems/activerecord-5.2.4.6/lib/active_record/connection_adapters/abstract_adapter.rb:580:in `block in log'
/seek/vendor/bundle/ruby/2.6.0/gems/activesupport-5.2.4.6/lib/active_support/notifications/instrumenter.rb:23:in `instrument'
/seek/vendor/bundle/ruby/2.6.0/gems/activerecord-5.2.4.6/lib/active_record/connection_adapters/abstract_adapter.rb:571:in `log'
/seek/vendor/bundle/ruby/2.6.0/gems/activerecord-5.2.4.6/lib/active_record/connection_adapters/abstract_mysql_adapter.rb:185:in `execute'
/seek/vendor/bundle/ruby/2.6.0/gems/activerecord-5.2.4.6/lib/active_record/connection_adapters/mysql/database_statements.rb:28:in `execute'
/seek/vendor/bundle/ruby/2.6.0/gems/activerecord-5.2.4.6/lib/active_record/connection_adapters/abstract/schema_statements.rb:311:in `create_table'
/seek/vendor/bundle/ruby/2.6.0/gems/activerecord-5.2.4.6/lib/active_record/migration.rb:871:in `block in method_missing'
/seek/vendor/bundle/ruby/2.6.0/gems/activerecord-5.2.4.6/lib/active_record/migration.rb:840:in `block in say_with_time'
/seek/vendor/bundle/ruby/2.6.0/gems/activerecord-5.2.4.6/lib/active_record/migration.rb:840:in `say_with_time'
/seek/vendor/bundle/ruby/2.6.0/gems/activerecord-5.2.4.6/lib/active_record/migration.rb:860:in `method_missing'
/seek/vendor/bundle/ruby/2.6.0/gems/activerecord-5.2.4.6/lib/active_record/migration/compatibility.rb:32:in `create_table'
/seek/vendor/bundle/ruby/2.6.0/gems/activerecord-5.2.4.6/lib/active_record/migration/compatibility.rb:73:in `create_table'
/seek/vendor/bundle/ruby/2.6.0/gems/activerecord-5.2.4.6/lib/active_record/migration/compatibility.rb:141:in `create_table'
/seek/db/migrate/20190329092301_add_table_human_diseases.rb:4:in `change'
/seek/vendor/bundle/ruby/2.6.0/gems/activerecord-5.2.4.6/lib/active_record/migration.rb:814:in `exec_migration'
/seek/vendor/bundle/ruby/2.6.0/gems/activerecord-5.2.4.6/lib/active_record/migration.rb:798:in `block (2 levels) in migrate'
/seek/vendor/bundle/ruby/2.6.0/gems/activerecord-5.2.4.6/lib/active_record/migration.rb:797:in `block in migrate'
/seek/vendor/bundle/ruby/2.6.0/gems/activerecord-5.2.4.6/lib/active_record/connection_adapters/abstract/connection_pool.rb:416:in `with_connection'
/seek/vendor/bundle/ruby/2.6.0/gems/activerecord-5.2.4.6/lib/active_record/migration.rb:796:in `migrate'
/seek/vendor/bundle/ruby/2.6.0/gems/activerecord-5.2.4.6/lib/active_record/migration.rb:977:in `migrate'
/seek/vendor/bundle/ruby/2.6.0/gems/activerecord-5.2.4.6/lib/active_record/migration.rb:1292:in `block in execute_migration_in_transaction'
/seek/vendor/bundle/ruby/2.6.0/gems/activerecord-5.2.4.6/lib/active_record/migration.rb:1345:in `ddl_transaction'
/seek/vendor/bundle/ruby/2.6.0/gems/activerecord-5.2.4.6/lib/active_record/migration.rb:1291:in `execute_migration_in_transaction'
/seek/vendor/bundle/ruby/2.6.0/gems/activerecord-5.2.4.6/lib/active_record/migration.rb:1263:in `block in migrate_without_lock'
/seek/vendor/bundle/ruby/2.6.0/gems/activerecord-5.2.4.6/lib/active_record/migration.rb:1262:in `each'
/seek/vendor/bundle/ruby/2.6.0/gems/activerecord-5.2.4.6/lib/active_record/migration.rb:1262:in `migrate_without_lock'
/seek/vendor/bundle/ruby/2.6.0/gems/activerecord-5.2.4.6/lib/active_record/migration.rb:1210:in `block in migrate'
/seek/vendor/bundle/ruby/2.6.0/gems/activerecord-5.2.4.6/lib/active_record/migration.rb:1363:in `with_advisory_lock'
/seek/vendor/bundle/ruby/2.6.0/gems/activerecord-5.2.4.6/lib/active_record/migration.rb:1210:in `migrate'
/seek/vendor/bundle/ruby/2.6.0/gems/activerecord-5.2.4.6/lib/active_record/migration.rb:1036:in `up'
/seek/vendor/bundle/ruby/2.6.0/gems/activerecord-5.2.4.6/lib/active_record/migration.rb:1011:in `migrate'
/seek/vendor/bundle/ruby/2.6.0/gems/activerecord-5.2.4.6/lib/active_record/tasks/database_tasks.rb:172:in `migrate'
/seek/vendor/bundle/ruby/2.6.0/gems/activerecord-5.2.4.6/lib/active_record/railties/databases.rake:60:in `block (2 levels) in <top (required)>'
/seek/lib/tasks/seek_upgrades.rake:38:in `block (2 levels) in <top (required)>'
/seek/vendor/bundle/ruby/2.6.0/gems/rake-13.0.3/exe/rake:27:in `<top (required)>'
/usr/local/bin/bundle:23:in `load'
/usr/local/bin/bundle:23:in `<main>'

Caused by:
ActiveRecord::StatementInvalid: Mysql2::Error: Tablespace '`seek_docker`.`human_diseases`' exists.: CREATE TABLE `human_diseases` (`id` int NOT NULL AUTO_INCREMENT PRIMARY KEY, `title` varchar(255), `doid_id` varchar(255), `created_at` datetime, `updated_at` datetime, `first_letter` varchar(255), `uuid` varchar(255)) ENGINE=InnoDB
/seek/vendor/bundle/ruby/2.6.0/gems/mysql2-0.5.3/lib/mysql2/client.rb:131:in `_query'
/seek/vendor/bundle/ruby/2.6.0/gems/mysql2-0.5.3/lib/mysql2/client.rb:131:in `block in query'
/seek/vendor/bundle/ruby/2.6.0/gems/mysql2-0.5.3/lib/mysql2/client.rb:130:in `handle_interrupt'
/seek/vendor/bundle/ruby/2.6.0/gems/mysql2-0.5.3/lib/mysql2/client.rb:130:in `query'
/seek/vendor/bundle/ruby/2.6.0/gems/activerecord-5.2.4.6/lib/active_record/connection_adapters/abstract_mysql_adapter.rb:187:in `block (2 levels) in execute'
/seek/vendor/bundle/ruby/2.6.0/gems/activesupport-5.2.4.6/lib/active_support/dependencies/interlock.rb:48:in `block in permit_concurrent_loads'
/seek/vendor/bundle/ruby/2.6.0/gems/activesupport-5.2.4.6/lib/active_support/concurrency/share_lock.rb:187:in `yield_shares'
/seek/vendor/bundle/ruby/2.6.0/gems/activesupport-5.2.4.6/lib/active_support/dependencies/interlock.rb:47:in `permit_concurrent_loads'
/seek/vendor/bundle/ruby/2.6.0/gems/activerecord-5.2.4.6/lib/active_record/connection_adapters/abstract_mysql_adapter.rb:186:in `block in execute'
/seek/vendor/bundle/ruby/2.6.0/gems/activerecord-5.2.4.6/lib/active_record/connection_adapters/abstract_adapter.rb:581:in `block (2 levels) in log'
/seek/vendor/bundle/ruby/2.6.0/gems/activerecord-5.2.4.6/lib/active_record/connection_adapters/abstract_adapter.rb:580:in `block in log'
/seek/vendor/bundle/ruby/2.6.0/gems/activesupport-5.2.4.6/lib/active_support/notifications/instrumenter.rb:23:in `instrument'
/seek/vendor/bundle/ruby/2.6.0/gems/activerecord-5.2.4.6/lib/active_record/connection_adapters/abstract_adapter.rb:571:in `log'
/seek/vendor/bundle/ruby/2.6.0/gems/activerecord-5.2.4.6/lib/active_record/connection_adapters/abstract_mysql_adapter.rb:185:in `execute'
/seek/vendor/bundle/ruby/2.6.0/gems/activerecord-5.2.4.6/lib/active_record/connection_adapters/mysql/database_statements.rb:28:in `execute'
/seek/vendor/bundle/ruby/2.6.0/gems/activerecord-5.2.4.6/lib/active_record/connection_adapters/abstract/schema_statements.rb:311:in `create_table'
/seek/vendor/bundle/ruby/2.6.0/gems/activerecord-5.2.4.6/lib/active_record/migration.rb:871:in `block in method_missing'
/seek/vendor/bundle/ruby/2.6.0/gems/activerecord-5.2.4.6/lib/active_record/migration.rb:840:in `block in say_with_time'
/seek/vendor/bundle/ruby/2.6.0/gems/activerecord-5.2.4.6/lib/active_record/migration.rb:840:in `say_with_time'
/seek/vendor/bundle/ruby/2.6.0/gems/activerecord-5.2.4.6/lib/active_record/migration.rb:860:in `method_missing'
/seek/vendor/bundle/ruby/2.6.0/gems/activerecord-5.2.4.6/lib/active_record/migration/compatibility.rb:32:in `create_table'
/seek/vendor/bundle/ruby/2.6.0/gems/activerecord-5.2.4.6/lib/active_record/migration/compatibility.rb:73:in `create_table'
/seek/vendor/bundle/ruby/2.6.0/gems/activerecord-5.2.4.6/lib/active_record/migration/compatibility.rb:141:in `create_table'
/seek/db/migrate/20190329092301_add_table_human_diseases.rb:4:in `change'
/seek/vendor/bundle/ruby/2.6.0/gems/activerecord-5.2.4.6/lib/active_record/migration.rb:814:in `exec_migration'
/seek/vendor/bundle/ruby/2.6.0/gems/activerecord-5.2.4.6/lib/active_record/migration.rb:798:in `block (2 levels) in migrate'
/seek/vendor/bundle/ruby/2.6.0/gems/activerecord-5.2.4.6/lib/active_record/migration.rb:797:in `block in migrate'
/seek/vendor/bundle/ruby/2.6.0/gems/activerecord-5.2.4.6/lib/active_record/connection_adapters/abstract/connection_pool.rb:416:in `with_connection'
/seek/vendor/bundle/ruby/2.6.0/gems/activerecord-5.2.4.6/lib/active_record/migration.rb:796:in `migrate'
/seek/vendor/bundle/ruby/2.6.0/gems/activerecord-5.2.4.6/lib/active_record/migration.rb:977:in `migrate'
/seek/vendor/bundle/ruby/2.6.0/gems/activerecord-5.2.4.6/lib/active_record/migration.rb:1292:in `block in execute_migration_in_transaction'
/seek/vendor/bundle/ruby/2.6.0/gems/activerecord-5.2.4.6/lib/active_record/migration.rb:1345:in `ddl_transaction'
/seek/vendor/bundle/ruby/2.6.0/gems/activerecord-5.2.4.6/lib/active_record/migration.rb:1291:in `execute_migration_in_transaction'
/seek/vendor/bundle/ruby/2.6.0/gems/activerecord-5.2.4.6/lib/active_record/migration.rb:1263:in `block in migrate_without_lock'
/seek/vendor/bundle/ruby/2.6.0/gems/activerecord-5.2.4.6/lib/active_record/migration.rb:1262:in `each'
/seek/vendor/bundle/ruby/2.6.0/gems/activerecord-5.2.4.6/lib/active_record/migration.rb:1262:in `migrate_without_lock'
/seek/vendor/bundle/ruby/2.6.0/gems/activerecord-5.2.4.6/lib/active_record/migration.rb:1210:in `block in migrate'
/seek/vendor/bundle/ruby/2.6.0/gems/activerecord-5.2.4.6/lib/active_record/migration.rb:1363:in `with_advisory_lock'
/seek/vendor/bundle/ruby/2.6.0/gems/activerecord-5.2.4.6/lib/active_record/migration.rb:1210:in `migrate'
/seek/vendor/bundle/ruby/2.6.0/gems/activerecord-5.2.4.6/lib/active_record/migration.rb:1036:in `up'
/seek/vendor/bundle/ruby/2.6.0/gems/activerecord-5.2.4.6/lib/active_record/migration.rb:1011:in `migrate'
/seek/vendor/bundle/ruby/2.6.0/gems/activerecord-5.2.4.6/lib/active_record/tasks/database_tasks.rb:172:in `migrate'
/seek/vendor/bundle/ruby/2.6.0/gems/activerecord-5.2.4.6/lib/active_record/railties/databases.rake:60:in `block (2 levels) in <top (required)>'
/seek/lib/tasks/seek_upgrades.rake:38:in `block (2 levels) in <top (required)>'
/seek/vendor/bundle/ruby/2.6.0/gems/rake-13.0.3/exe/rake:27:in `<top (required)>'
/usr/local/bin/bundle:23:in `load'
/usr/local/bin/bundle:23:in `<main>'

Caused by:
Mysql2::Error: Tablespace '`seek_docker`.`human_diseases`' exists.
/seek/vendor/bundle/ruby/2.6.0/gems/mysql2-0.5.3/lib/mysql2/client.rb:131:in `_query'
/seek/vendor/bundle/ruby/2.6.0/gems/mysql2-0.5.3/lib/mysql2/client.rb:131:in `block in query'
/seek/vendor/bundle/ruby/2.6.0/gems/mysql2-0.5.3/lib/mysql2/client.rb:130:in `handle_interrupt'
/seek/vendor/bundle/ruby/2.6.0/gems/mysql2-0.5.3/lib/mysql2/client.rb:130:in `query'
/seek/vendor/bundle/ruby/2.6.0/gems/activerecord-5.2.4.6/lib/active_record/connection_adapters/abstract_mysql_adapter.rb:187:in `block (2 levels) in execute'
/seek/vendor/bundle/ruby/2.6.0/gems/activesupport-5.2.4.6/lib/active_support/dependencies/interlock.rb:48:in `block in permit_concurrent_loads'
/seek/vendor/bundle/ruby/2.6.0/gems/activesupport-5.2.4.6/lib/active_support/concurrency/share_lock.rb:187:in `yield_shares'
/seek/vendor/bundle/ruby/2.6.0/gems/activesupport-5.2.4.6/lib/active_support/dependencies/interlock.rb:47:in `permit_concurrent_loads'
/seek/vendor/bundle/ruby/2.6.0/gems/activerecord-5.2.4.6/lib/active_record/connection_adapters/abstract_mysql_adapter.rb:186:in `block in execute'
/seek/vendor/bundle/ruby/2.6.0/gems/activerecord-5.2.4.6/lib/active_record/connection_adapters/abstract_adapter.rb:581:in `block (2 levels) in log'
/seek/vendor/bundle/ruby/2.6.0/gems/activerecord-5.2.4.6/lib/active_record/connection_adapters/abstract_adapter.rb:580:in `block in log'
/seek/vendor/bundle/ruby/2.6.0/gems/activesupport-5.2.4.6/lib/active_support/notifications/instrumenter.rb:23:in `instrument'
/seek/vendor/bundle/ruby/2.6.0/gems/activerecord-5.2.4.6/lib/active_record/connection_adapters/abstract_adapter.rb:571:in `log'
/seek/vendor/bundle/ruby/2.6.0/gems/activerecord-5.2.4.6/lib/active_record/connection_adapters/abstract_mysql_adapter.rb:185:in `execute'
/seek/vendor/bundle/ruby/2.6.0/gems/activerecord-5.2.4.6/lib/active_record/connection_adapters/mysql/database_statements.rb:28:in `execute'
/seek/vendor/bundle/ruby/2.6.0/gems/activerecord-5.2.4.6/lib/active_record/connection_adapters/abstract/schema_statements.rb:311:in `create_table'
/seek/vendor/bundle/ruby/2.6.0/gems/activerecord-5.2.4.6/lib/active_record/migration.rb:871:in `block in method_missing'
/seek/vendor/bundle/ruby/2.6.0/gems/activerecord-5.2.4.6/lib/active_record/migration.rb:840:in `block in say_with_time'
/seek/vendor/bundle/ruby/2.6.0/gems/activerecord-5.2.4.6/lib/active_record/migration.rb:840:in `say_with_time'
/seek/vendor/bundle/ruby/2.6.0/gems/activerecord-5.2.4.6/lib/active_record/migration.rb:860:in `method_missing'
/seek/vendor/bundle/ruby/2.6.0/gems/activerecord-5.2.4.6/lib/active_record/migration/compatibility.rb:32:in `create_table'
/seek/vendor/bundle/ruby/2.6.0/gems/activerecord-5.2.4.6/lib/active_record/migration/compatibility.rb:73:in `create_table'
/seek/vendor/bundle/ruby/2.6.0/gems/activerecord-5.2.4.6/lib/active_record/migration/compatibility.rb:141:in `create_table'
/seek/db/migrate/20190329092301_add_table_human_diseases.rb:4:in `change'
/seek/vendor/bundle/ruby/2.6.0/gems/activerecord-5.2.4.6/lib/active_record/migration.rb:814:in `exec_migration'
/seek/vendor/bundle/ruby/2.6.0/gems/activerecord-5.2.4.6/lib/active_record/migration.rb:798:in `block (2 levels) in migrate'
/seek/vendor/bundle/ruby/2.6.0/gems/activerecord-5.2.4.6/lib/active_record/migration.rb:797:in `block in migrate'
/seek/vendor/bundle/ruby/2.6.0/gems/activerecord-5.2.4.6/lib/active_record/connection_adapters/abstract/connection_pool.rb:416:in `with_connection'
/seek/vendor/bundle/ruby/2.6.0/gems/activerecord-5.2.4.6/lib/active_record/migration.rb:796:in `migrate'
/seek/vendor/bundle/ruby/2.6.0/gems/activerecord-5.2.4.6/lib/active_record/migration.rb:977:in `migrate'
/seek/vendor/bundle/ruby/2.6.0/gems/activerecord-5.2.4.6/lib/active_record/migration.rb:1292:in `block in execute_migration_in_transaction'
/seek/vendor/bundle/ruby/2.6.0/gems/activerecord-5.2.4.6/lib/active_record/migration.rb:1345:in `ddl_transaction'
/seek/vendor/bundle/ruby/2.6.0/gems/activerecord-5.2.4.6/lib/active_record/migration.rb:1291:in `execute_migration_in_transaction'
/seek/vendor/bundle/ruby/2.6.0/gems/activerecord-5.2.4.6/lib/active_record/migration.rb:1263:in `block in migrate_without_lock'
/seek/vendor/bundle/ruby/2.6.0/gems/activerecord-5.2.4.6/lib/active_record/migration.rb:1262:in `each'
/seek/vendor/bundle/ruby/2.6.0/gems/activerecord-5.2.4.6/lib/active_record/migration.rb:1262:in `migrate_without_lock'
/seek/vendor/bundle/ruby/2.6.0/gems/activerecord-5.2.4.6/lib/active_record/migration.rb:1210:in `block in migrate'
/seek/vendor/bundle/ruby/2.6.0/gems/activerecord-5.2.4.6/lib/active_record/migration.rb:1363:in `with_advisory_lock'
/seek/vendor/bundle/ruby/2.6.0/gems/activerecord-5.2.4.6/lib/active_record/migration.rb:1210:in `migrate'
/seek/vendor/bundle/ruby/2.6.0/gems/activerecord-5.2.4.6/lib/active_record/migration.rb:1036:in `up'
/seek/vendor/bundle/ruby/2.6.0/gems/activerecord-5.2.4.6/lib/active_record/migration.rb:1011:in `migrate'
/seek/vendor/bundle/ruby/2.6.0/gems/activerecord-5.2.4.6/lib/active_record/tasks/database_tasks.rb:172:in `migrate'
/seek/vendor/bundle/ruby/2.6.0/gems/activerecord-5.2.4.6/lib/active_record/railties/databases.rake:60:in `block (2 levels) in <top (required)>'
/seek/lib/tasks/seek_upgrades.rake:38:in `block (2 levels) in <top (required)>'
/seek/vendor/bundle/ruby/2.6.0/gems/rake-13.0.3/exe/rake:27:in `<top (required)>'
/usr/local/bin/bundle:23:in `load'
/usr/local/bin/bundle:23:in `<main>'
Tasks: TOP => db:migrate
(See full trace by running task with --trace)
```

**Server**:
* Dockerized? yes
* Version 1.10.3

**Additional context**

When trying to upgrade to version 1.11.1 a few weeks back I got another error:

```
$ docker exec -it seek docker/upgrade.sh
... Setting version visibility...
  Updating 230 DataFile::Version's visibilityrake aborted!
NoMethodError: undefined method `latest_version' for nil:NilClass
/seek/lib/seek/explicit_versioning.rb:69:in `latest_version?'
/seek/lib/tasks/seek_upgrades.rake:140:in `block (5 levels) in <top (required)>'
/seek/vendor/bundle/ruby/2.6.0/gems/activerecord-5.2.4.6/lib/active_record/relation/batches.rb:70:in `block (2 levels) in find_each'
/seek/vendor/bundle/ruby/2.6.0/gems/activerecord-5.2.4.6/lib/active_record/relation/batches.rb:70:in `each'
/seek/vendor/bundle/ruby/2.6.0/gems/activerecord-5.2.4.6/lib/active_record/relation/batches.rb:70:in `block in find_each'
/seek/vendor/bundle/ruby/2.6.0/gems/activerecord-5.2.4.6/lib/active_record/relation/batches.rb:136:in `block in find_in_batches'
/seek/vendor/bundle/ruby/2.6.0/gems/activerecord-5.2.4.6/lib/active_record/relation/batches.rb:238:in `block in in_batches'
/seek/vendor/bundle/ruby/2.6.0/gems/activerecord-5.2.4.6/lib/active_record/relation/batches.rb:222:in `loop'
/seek/vendor/bundle/ruby/2.6.0/gems/activerecord-5.2.4.6/lib/active_record/relation/batches.rb:222:in `in_batches'
/seek/vendor/bundle/ruby/2.6.0/gems/activerecord-5.2.4.6/lib/active_record/relation/batches.rb:135:in `find_in_batches'
/seek/vendor/bundle/ruby/2.6.0/gems/activerecord-5.2.4.6/lib/active_record/relation/batches.rb:69:in `find_each'
/seek/lib/tasks/seek_upgrades.rake:139:in `block (4 levels) in <top (required)>'
/seek/lib/tasks/seek_upgrades.rake:127:in `each'
/seek/lib/tasks/seek_upgrades.rake:127:in `block (3 levels) in <top (required)>'
/seek/lib/object_extensions.rb:24:in `disable_authorization_checks'
/seek/lib/tasks/seek_upgrades.rake:126:in `block (2 levels) in <top (required)>'
/seek/lib/tasks/seek_upgrades.rake:47:in `block (2 levels) in <top (required)>'
/seek/vendor/bundle/ruby/2.6.0/gems/rake-13.0.3/exe/rake:27:in `<top (required)>'
/usr/local/bin/bundle:23:in `load'
/usr/local/bin/bundle:23:in `<main>'
Tasks: TOP => seek:upgrade_version_tasks => seek:set_version_visibility
(See full trace by running task with --trace)
```

I backed up the database and filestore previously so I can restore the server.

Thank you for your help.

**Comment by Linus4** [comment]
Hello @stuzart , I followed your instructions and with the cleaned-up database I get the following error when running `docker/upgrade.sh`.

```
== 20191120162006 CreateCustomMetadataType: migrating =========================
-- table_exists?(:custom_metadata_types)
   -> 0.0005s
-- drop_table(:custom_metadata_types)
rake aborted!
StandardError: An error has occurred, all later migrations canceled:

Mysql2::Error: Unknown table 'seek_docker.custom_metadata_types': DROP TABLE `custom_metadata_types`
```

Both the `custom_metadata_types.ibd` and the `custom_metadata_types.frm` files exist in the database volume in `/var/lib/mysql/seek_docker` before running the script. After the error occurs `custom_metadata_types.ibd` still exists, but `custom_metadata_types.frm` is gone.

I deleted the `.ibd` file and ran the upgrade script again. I got the same error for `custom_metadata_attributes` and `custom_metadata`, so I also deleted these files and ran the upgrade script. At that point, the upgrade ran until it hit error 1 from my initial post.

Is it okay to delete `custom_metadata`, `custom_metadata_types` and `custom_metadata_attributes` (`.ibd` and `.frm`)?
